identifier/package.json

```diff
@@ -50,6 +50,7 @@
     "@typescript-eslint/eslint-plugin": "^6.11.0",
     "@typescript-eslint/parser": "^6.9.0",
     "@typescript-eslint/typescript-estree": "^6.11.0",
+    "@vitejs/plugin-legacy": "^5.3.2",
```

Comment:
This plugin version is too new for the used vita. Please use a version which is for vite 4. Triggers warning:
```
➤ YN0060: │ vite is listed by your project with version 4.5.2, which doesn't satisfy what @vitejs/plugin-legacy (p6a866) requests (^5.0.0).
```

Reply:
Updated the version from 5.3.2 ( which bumped up the peer dependencies requirement of vite to 5, used 4.1.1 as it is the latest to use 4.5.2 of vite. Please check the change log of legacy plugin for better understanding. 

https://github.com/vitejs/vite/blob/main/packages/plugin-legacy/CHANGELOG.md